lib/emoji.js

```diff
@@ -390,7 +390,8 @@
 			}else if (self.use_css_imgs){
 				return '<span class="emoji emoji-'+idx+'"'+title+'>'+text+'</span>'+extra;
 			}else if (self.image_mode){
-				return '<img class="emoji emoji-'+idx+'"'+title+' src="'+img+'">'+extra;
+				var unicode = '0x'+idx;
```

Comment:
`String.fromCodePoint()` won't parse correctly unless `0x` is added before the unicode string. That pattern breaks other modes, thus the need for single use here.